easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.7-iomkl-2018a-R-3.5.0.eb

```diff
@@ -0,0 +1,938 @@
+    }),
+    ('doRNG', '1.7.1', {
+        'checksums': ['27533d54464889d1c21301594137fc0f536574e3a413d61d7df9463ab12a67e9'],
+    }),
+    ('bumphunter', '1.22.0', {
+        'checksums': ['bedff4dc62c5274ac835c877de825f959d1f40e0779d356d42117847efcd4b34'],
+    }),
+    ('survival', '2.42-3', {
+        'checksums': ['a07732d7717f7a01543d95cf60cdc7cb5f46544cb7297ddd2a59bcfaddd2ea96'],
+    }),
+    ('multtest', '2.36.0', {
+        'checksums': ['e0d2f003b0afd674fb4f16255b20afa146491e65442a05511f81d337054c2485'],
+    }),
+    ('siggenes', '1.54.0', {
+        'checksums': ['1e9b945bfdcefbe5357f2d7169af27d4f4d45a90842d623488c1a489909d2f5c'],
+    }),
+    ('DynDoc', '1.58.0', {
+        'checksums': ['78572bbef190ba20cfa7c9a23f62570aebe35b73e83e18c590875c6fc49c4dd2'],
+    }),
+    ('genoset', '1.36.0', {
+        'checksums': ['6313ba101617a192ffc7066b3fccf4b564760f918838c5b53cb53e3faf3d33b8'],
+    }),
+    ('NOISeq', '2.24.0', {
+        'checksums': ['cb151bea959fbafa2a96a24ba507f158f740d66c2da5e306efd78fbf268a522a'],
+    }),
+    ('Rgraphviz', '2.24.0', {
+        'checksums': ['7ee16deb1c30f60bb5ad1382b99006b2472be61c9523e40d37c4b2dedd876780'],
+    }),
+    ('RNASeqPower', '1.20.0', {
+        'checksums': ['1ee658caaef9d9e4fd8a1085e459b318438d763f0e271a45fb941d5ea6e45630'],
+    }),
+    ('annotate', '1.58.0', {
+        'checksums': ['4e3234694c5f1b2713b64e0e548e1e9a4d21d02d24e638280dea90e0b767b6e2'],
+    }),
+    ('GSEABase', '1.42.0', {
+        'checksums': ['ead00709a662c67665f4c176df3b32789118ce3269cb730c4e69ccb8b5487b85'],
+    }),
+    ('genefilter', '1.62.0', {
+        'checksums': ['11ab26d789c6ee644545d5b371a263fc4dbacb839f850a64e7b25e2d80738092'],
+    }),
+    ('Category', '2.46.0', {
+        'checksums': ['a0bf452fcfd0513f3e46a98880e1af03b783e7ad82fe2346d5fcb5d090c28e0f'],
+    }),
+    ('GOstats', '2.46.0', {
+        'checksums': ['00b769e3bee17a76d9962461594a8b82da55610032658cc414bca4567ef3b5c4'],
+    }),
+    ('BSgenome', '1.48.0', {
+        'checksums': ['e815eb2def62b70e9aade10094f0cee3d9d3c44179581a4a9bf7b68271bc62e6'],
+    }),
+    ('VariantAnnotation', '1.26.0', {
+        'checksums': ['c7b26a281105495790b0fda335f07896d3340a61af42364c3d4aee535c636c19'],
+    }),
+    ('htmltools', '0.3.6', {
+        'checksums': ['44affb82f9c2fd76c9e2b58f9229adb003217932b68c3fdbf1327c8d74c868a2'],
+    }),
+    ('later', '0.7.3', {
+        'checksums': ['5c57148c255ee5da61cb748550b73bad7094f4e045b99b362473be5e3dbc4212'],
+    }),
+    ('promises', '1.0.1', {
+        'checksums': ['c2dbc7734adf009377a41e570dfe0d82afb91335c9d0ca1ef464b9bdcca65558'],
+    }),
+    ('httpuv', '1.4.4.1', {
+        'checksums': ['c6a44f262df83bae0fcf808a07c0b3da5f0bc9ea29aa7c6eccf421d541c07c8a'],
+    }),
+    ('sourcetools', '0.1.7', {
+        'checksums': ['47984406efb3b3face133979ccbae9fefb7360b9a6ca1a1c11473681418ed2ca'],
+    }),
+    ('shiny', '1.1.0', {
```

Comment:
@edmondac There are a whole bunch of additional R libraries included in here... Why is that necessary, these are already provided as a part of the R 3.5.0 dependency?

Reply:
Hmm. I didn't add them deliberately. Presumably easy_update added them.

Reply:
I'll remove them